4.analyze_data/notebooks/UMAP/scripts/0.UMAP_coordinates.py

```diff
@@ -156,18 +156,6 @@
     if plate_name in selected_plates:
         df = pd.read_parquet(file_path)
 
-        # Update Metadata_Plate for Plate_3_prime
-        if plate_name == "Plate_3_prime":
-            df["Metadata_Plate"] = "Plate_3_prime"
```

Comment:
Is this being removed because of the NF1 bug fix applied earlier?

Reply:
Yup! Bug is fixed so no need to include it anymore. This piece of code is still probably around in other places that might need to be refactored or not, doesn't really hurt much.